src/test/java/teammates/test/cases/browsertests/InstructorCourseEnrollPageUiTest.java

```diff
@@ -96,8 +96,10 @@ private void testEnrollAction() throws Exception {
                 .withUserId(testData.instructors.get("CCEnrollUiT.teammates.test").googleId)
                 .withCourseId(courseId);
         InstructorCoursesDetailsPage detailsPage =
-                loginAdminToPage(coursesPageUrl, InstructorCoursesDetailsPage.class);
-        assertEquals(6, detailsPage.getStudentCountForCourse("CCEnrollUiT.CS2104"));
+
+                loginAdminToPage(browser, coursesPageUrl, InstructorCoursesDetailsPage.class);
+        assertEquals(6, detailsPage.getStudentCountForCourse());
```

Comment:
Your build log says **cannot instantiate from arguments because actual and formal argument lists differ in length**.

Any reasons why you have added the extra parameter to `loginAdminToPage` method ?.